src/screens/Settings/index.js

```diff
@@ -83,11 +83,15 @@ const Settings = () => {
             text="Websocket Research"
             onPress={() => navigation.navigate('WebsocketResearchScreen')}
           /> */}
-          <ProfileSettingItem text="Delete Account" onPress={showDeleteAccountAlert} />
-          <ProfileSettingItem text="Logout" onPress={doLogout} />
+          <ProfileSettingItem
+            testID="delete"
+            text="Delete Account"
+            onPress={showDeleteAccountAlert}
+          />
+          <ProfileSettingItem testID="logout" text="Logout" onPress={doLogout} />
         </View>
         <View>
-          <TouchableOpacity onPress={turnOnDebugMode}>
+          <TouchableOpacity testID="debugmode" onPress={turnOnDebugMode}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's a good practice to add `testID` for testing purposes. However, ensure that the function `turnOnDebugMode` has proper error handling and doesn't crash the app in case of any exceptions.
```diff
- <TouchableOpacity testID="debugmode" onPress={turnOnDebugMode}>
+ <TouchableOpacity testID="debugmode" onPress={() => {
+   try {
+     turnOnDebugMode();
+   } catch (error) {
+     console.error('Error turning on debug mode:', error);
+   }
+ }}>
```
This will help to avoid unhandled promise rejections which can lead to hard crashes in production.


<!-- This is an auto-generated comment by OSS CodeRabbit -->